scripts/timing.py

```diff
@@ -190,6 +205,34 @@ def write_file(n: int):
     global TOUCHED_FILES
     TOUCHED_FILES.append(name)
 
+def write_fe_token() -> str:
+    now = str(datetime.datetime.now())
+    call_or_error(['sed', '-e', 's/^.*timing.py.*$/    timing.py {}/'.format(now), '-i', BLORG_FRONTEND_INDEX])
+    return now
+
+def wait_for_fe_token(token: str):
+    url = fe_url()
+    print('Waiting for token to appear: {}'.format(url))
+
+    found = False
+    while not found:
+        out = curl(url)
+        found = out.find(token) != -1
```

Comment:
or `if token not in out` -- saves you a line

Reply:
good point!

Reply:
oh wait, this doesn't work, b/c you need the `found` variable elsewhere in the function

Reply:
Ha you're right nvm